core/modules/dashboard/dashboard.module

```diff
@@ -222,7 +222,7 @@ function dashboard_preprocess_layout(&$variables) {
 function dashboard_preprocess_block(&$variables) {
   /* @var Layout $layout */
   $layout = $variables['layout'];
-  if ($layout->hasContexts(array('dashboard'))) {
+  if (!is_null($layout) && $layout->hasContexts(array('dashboard'))) {
```

Comment:
```suggestion
  if (is_a($layout, 'Layout') && $layout->hasContexts(array('dashboard'))) {
```

Reply:
@olafgrabienski I've added the suggested change. It is a core PHP function https://www.php.net/manual/en/function.is-a.php